tests/phel/test/html.phel

```diff
@@ -37,56 +36,38 @@
 
 (deftest arrays
   (is (= "<p>a</p><p>b</p>" (html [:p "a"] [:p "b"])))
-  (is (= "<body>foobar</body>" (html [:body @["foo" "bar"]])))
-  (is (= "<p>a</p><p>b</p>" (html @[[:p "a"] [:p "b"]]))))
+  # (is (= "<body>foobar</body>" (html [:body ["foo" "bar"]])))
+  # (is (= "<p>a</p><p>b</p>" (html [[:p "a"] [:p "b"]])))
```

Comment:
Why is this commented? :)

Reply:
Well, so far I can see, this `(is (= "<body>foobar</body>" (html [:body ["foo" "bar"]])))` is not working :) 
<img width="1188" alt="Screenshot 2021-12-22 at 14 29 34" src="https://user-images.githubusercontent.com/5256287/147100128-31c5c49e-ce7c-4055-9710-bcc3605709d5.png">

@jenshaase do we want to fix this before merging this PR? or could we do that in a follow up PR and that's why you just commented it out?

Reply:
Ah thanks for the reminder. I wanted to check it later but forgot about it. I now fixed the tests.